website/profile/views.py

```diff
@@ -382,6 +383,39 @@ def user_notifications(auth, **kwargs):
         'mailing_lists': auth.user.mailing_lists
     }
 
+@dev_only
+@must_be_logged_in
+def oauth_application_list(auth, **kwargs):
+    """Return app creation page with list of known apps"""
+    # TODO: Hardcoded URL
```

Comment:
It's still hardcoding the structure of the URL, even if it's not hardcoding the base route. I don't know if we have any system of consistently identifying routes that referenced by something other than views.